src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,17 @@
+import * as dateFns from 'date-fns'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+
+const getTime = (amntOfSeconds: number): string => {
+  const HHmmss = getFormattedDate(
+    String(dateFns.addSeconds(new Date(0), amntOfSeconds)),
+    DateFormatType.HOURS_MINUTES_SECONDS
+  )
+  const splittedTime = HHmmss.split(':')
+  let time = splittedTime[0] + ':' + splittedTime[1] + ':' + splittedTime[2]
```

Comment:
It looks very confused. Let's investigate how we can do it by another way. Maybe with `date-fns`

Reply:
This function "getTime" transform string in format "hh:mm:ss", and then analyse, if time has not "hh", cut this part to avoid extra characters. In "date-fns" i don't find this functionality.